### ISSUE TITLE
Visual Communication of Rank Counters

### ISSUE THREAD

**Issue by uqrs** [ppy/osu-web#3964]
In the new userpage web design- the Global Ranking and Country Ranking numbers might benefit from being preceded by an octotrophe (#)- yes, it does say "Ranking", but considering the fact # already has a very strong association with a rank, I feel like including it would be a very handy way to more clearly and quickly communicate the fact that this number indicates a players' rank.
![Rank Counter](https://cdn.discordapp.com/attachments/350554887441874946/520606149230329860/rank.png)

**Comment by peppy** [comment]
I’m neutral on this one, but it is what people are used to I guess.

**Comment by arflyte** [comment]
Yeah. I have to agree on this. Let's add the hash back into the ranking. Reduce the size of the font to make it fit.
